ModularTegustation/tegu_items/gadgets/unpowered.dm

```diff
@@ -499,3 +499,59 @@
 	)
 	mid_length = 2 SECONDS
 	volume = 20
+
+//Clerkbot Spawner
+/obj/item/clerkbot_gadget
+	name = "Instant Clerkbot Constructor"
+	desc = "An instant constructor for Clerkbots. Loyal little things that attack hostile creatures. In order to prevent \
+		abnormalities infesting the clerkbots, only those registered as a Lobotomy Corp clerk can activate them."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "clerkbot2_deactivated"
+
+/obj/item/clerkbot_gadget/attack_self(mob/user)
+	..()
+	if(!istype(user) || !(user?.mind?.assigned_role in GLOB.service_positions))
+		to_chat(user, span_notice("The Gadget's light flashes red. You aren't a clerk. Check the label before use."))
+		return
+	new /mob/living/simple_animal/hostile/clerkbot(get_turf(user))
+	to_chat(user, span_nicegreen("The Gadget turns warm and sparks."))
+	qdel(src)
+
+/mob/living/simple_animal/hostile/clerkbot/Initialize()
+	..()
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "clerkbot2"
+	icon_living = "clerkbot2"
+	if(prob(50))
+		icon_state = "clerkbot1"
+		icon_living = "clerkbot1"
```

Comment:
Move below object definition and remove obsolete code (stuff outside of prob)

Reply:
Done!